core/src/main/java/com/scalar/db/api/Operation.java

```diff
@@ -83,6 +83,7 @@ public Optional<String> forFullTableName() {
    * @param namespace target namespace for this operation
    * @return this object
    */
+  @Deprecated
```

Comment:
Ok, should I put the same deprecated version and removed version as the other methods in operation classes ?
```@Deprecated As of release 3.6.0. Will be removed in release 5.0.0```

Reply:
Without adding the `@Deprecated As of release 3.6.0. Will be removed in release 5.0.0` to the overridden methods Javadoc; this specific Javadoc field does not seem to be inherited from the root method and won't be shown on the overridden method Javadoc. 
Is it ok not to add it?

Reply:
Oh okay. I didn't know that. Maybe we need to use `{@inheritdoc}` for overridden methods, right? Or even if we use `{@inheritdoc}`, the `@deprecated` won't be shown on the overridden method Javadoc?

Reply:
`{@inheritdoc}` does not seem to work for the `@deprecated` field. 
Though, by adding only ```/** @deprecated As of release 3.6.0. Will be removed in release 5.0.0 */``` as Javadoc comment to the overridden methods, the Javadoc will show the inherited fields automatically and augmented with the `@deprecated` information.

![Screen Shot 2022-06-15 at 12 25 04](https://user-images.githubusercontent.com/3835021/173730358-a7ab0c56-fdda-478d-8ec7-a9418256cd7d.png)

Reply:
Thank you for the investigation. Okay, so let's add `@deprecated` javadoc to the overridden methods.

Reply:
One last thing, on each operation mutating deprecated method, I think we can redirect to the builder method as well. 
Though, instead of redirecting to the exact method below like the following because the user might wonder how to obtain a `com.scalar.db.api.builder.PutBuilder.Namespace` object.
```
/**
  * @deprecated As of release 3.6.0. Will be removed in release 5.0.0. Use the setter method of the Put builder instead, use {@link com.scalar.db.api.builder.PutBuilder.Namespace#namespace(String)}
  */
```
Writing it like below this gets the point across. 
```
/**
   * @deprecated As of release 3.6.0. Will be removed in release 5.0.0. Use the setter method of the
   *     Put builder instead; to create a Put builder, use {@link Put#newBuilder()}
   */
```
What do you think ?

Reply:
Sounds good. Thank you.

Reply:
Revised in https://github.com/scalar-labs/scalardb/pull/601/commits/0fbaed49b34872454ed6dd5ba2ea01e26b1be963